### ISSUE TITLE
AttributeError: 'GenericDict' object has no attribute 'pool'

### ISSUE THREAD

**Issue by Sarimuko** [locuslab/perturbation_learning#2]
Hi,

I have tried the command 'python robust_train.py --config-robust configs_robust/mi_unet_cvae_aug.json' and got the error 'AttributeError: 'GenericDict' object has no attribute 'pool''.

Could you please check the command?

Thanks!

**Comment by Sarimuko** [comment]
And the argument 'config-attack' for robust_train.py is missing while it is used in all.sh.

**Comment by riceric22** [comment]
Hi @Sarimuko! 

Thanks for reaching out! I've pushed the fixes to the repo in a56cb3cb0da84c76777dc0242ca150af6c2e42c6, but in summary: 

1. You can add the missing pool argument as follows: 
```
      "model": {
          "type": "mi_unet", 
          "output_distribution": "gaussian", 
          "chs": [64, 128, 256, 512, 1024], 
          "latent_dim": [128, 64, 32, 16, 16],
          "dim": [125, 187], 
          "bilinear": true, 
          "pool": [[125,187], 
                  [62, 93],
                  [31, 46],
                  [15, 23],
                  [7, 11]]
      },
```
to the model parameter in the config file (these are the pooling sizes for the MIP5 resolution). I had added this argument after running these experiments to handle the larger resolutions but hadn't updated the configs for the smaller resolutions, thanks for pointing this out! 

2. In the `all.sh` script, these calls with `config-attack` should be calling `robust_eval.py` instead of `robust_train.py`. Thanks for bringing this out as well!
